Kwc/Abstract.php

```diff
@@ -135,7 +135,7 @@ public static function getChildComponentClasses($class, $select = array())
             return array_unique($ret);
 
         } else if ($selectType == 'array' && is_string($class) && count($select) == 1 &&
-            isset($select['componentClass']) && count($select['componentClass']) == 1
+            isset($select['componentClass']) && is_string($select['componentClass'])
```

Comment:
leerer String wäre in dem Fall auch true, würde auch die Logik ändern?

Reply:
Nein, count("") == true